examples/permalink3d.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
```

Comment:
IMO, there are already too many examples in ngeo. Instead of creating a new one, could you add permalink support to the simple3d example?

Reply:
Ok i removed it